### ISSUE TITLE
Error Message "TLS initialization failed"

### ISSUE THREAD

**Issue by DStalkerBR** [crow-translate/crow-translate#28]
OS: Windows 10 Pro x64
Version: Crow-Translate-2.1.0-win64

When I try to translate something using Google or Yandex the message  "TLS initialization failed" show on the output instead of the translation.

![image](https://user-images.githubusercontent.com/34558728/56059941-d0db5700-5d3b-11e9-8207-46c8907d8449.png)

**Comment by ShaturIQ** [comment]
Does this problem occur every time?

**Comment by Shatur95** [comment]
Does this problem occur every time?

**Comment by DStalkerBR** [comment]
Yes, with both Google and Yandex, only Bing is working properly for me.

**Comment by Shatur95** [comment]
What is your Windows revision?

**Comment by DStalkerBR** [comment]
> What is your Windows revision?

Version 1809 (Build 17763.379)

**Comment by Shatur95** [comment]
I have 1809 too, but everything works as well. Have you tried a clean install?

Yandex and Google use GET requests, but Bing uses POST. It looks like POST requests are not working. Maybe you use a proxy?

**Comment by DStalkerBR** [comment]
> I have 1809 too, but everything works as well. Have you tried a clean install?

I made some modifications after I installed windows, but I asked a friend who did a clean install to test the translator and he's having the same problem.

> Yandex and Google use GET requests, but Bing uses POST. It looks like POST requests are not working. 

Since Bing is the engine which is actually working, wouldn't be the GET requests that aren't working?

>Maybe you use a proxy?

I'm not using a proxy, although I did try to use it to check if there was any change.

**Comment by Shatur95** [comment]
> Since Bing is the engine which is actually working, wouldn't be the GET requests that aren't working?
> I'm not using a proxy, although I did try to use it to check if there was any change.

This may be due to your provider... Try using a proxy. For example, you can download the [Tor Browser](https://www.torproject.org/download/) and configure it using Crow Translate or use [OpenVPN](https://openvpn.net/community-downloads/).

**Comment by DStalkerBR** [comment]
For some reason now Google and Yandex are working for me(without the VPN), but my friend is still having the same problem.

**Comment by Shatur95** [comment]
It's very strange :) Maybe you changed something in the connection settings?

**Comment by DStalkerBR** [comment]
I think I found out what caused the problem: The absence of **English language package** in the OS.
Both on my PC and on the other PC that had the problem the OS didn't had the english language  language package installed.
A few days before it worked correctly on my PC I had installed the en_US language pack, and after a while I did the same on my friend's PC to check it out, and soon after installing, the Google and Yandex engines worked properly.
I didn't suspect this before because even after I uninstalled the language package (en_US) both the engines were still working.
